src/peerbook/libp2p_peerbook.erl

```diff
@@ -505,6 +458,11 @@ terminate(_Reason, _State) ->
 %% Internal
 %%
 
+eligible_gossip_peer(Peer) ->
```

Comment:
So the goal here is to deliver some dialable and reasonably well connected peers from the peerbook?

Reply:
the idea was to replicate the existing functionality without tick and without storing stuff in the state no fold, etc).